From 00fb7cf5274ec9a62a440135ee26d23522e4a024 Mon Sep 17 00:00:00 2001
From: Viacheslav Bocharov <adeep@lexina.in>
Date: Tue, 18 Jun 2024 12:14:58 +0300
Subject: [PATCH 10/10] enable-DT-overlays-support

---
 arch/arm/Kconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/Kconfig b/arch/arm/Kconfig
index d812685c984..64df49e1622 100644
--- a/arch/arm/Kconfig
+++ b/arch/arm/Kconfig
@@ -803,6 +803,8 @@ config ARCH_OMAP2PLUS
 config ARCH_MESON
 	bool "Amlogic Meson"
 	select GPIO_EXTRA_HEADER
+	select OF_LIBFDT
+	select OF_LIBFDT_OVERLAY
 	imply DISTRO_DEFAULTS
 	imply DM_RNG
 	help
-- 
2.45.2

